Makefile

```diff
@@ -1,5 +1,17 @@
 GO := go
 GOFMT := gofmt
+PROTOC_INSTALLED := $(shell which protoc 2>/dev/null)
+PROTOC_EXPECTED_VERSION =libprotoc 3.21.12
+PROTOC_CURR_VERSION := $(shell protoc --version 2>/dev/null)
+BUF_INSTALLED := $(shell which buf 2>/dev/null)
+BUF_EXPECTED_VERSION = 1.15.0
+BUF_CURR_VERSION := $(shell buf --version 2>/dev/null)
```

Comment:
perhaps. assure me and I happily remove it :)

Reply:
That's a bigger ask! I can build up the fedora a bit more when I get the chance and see if the "native" versions work, but I will need to hack the Makefile to get there.

Or... it might just be easier to add some comments in the Makefile saying, "if you have any problems, please use these known good versions: ..."

Otherwise, Will and others may not be able to use the make target without building protoc, etc. from source, or grabbing some non-vanilla package.

Reply:
see https://github.com/RedHatInsights/authz/pull/41#discussion_r1124336666